command/genesis/polybft_params.go

```diff
@@ -551,9 +595,15 @@ func (p *genesisParams) getValidatorAccounts() ([]*validator.GenesisValidator, e
 
 			addr := types.StringToAddress(trimmedAddress)
 
-			stake, exists := p.stakeInfos[addr]
-			if !exists {
-				stake = command.DefaultStake
+			stake := big.NewInt(0)
+
+			if p.nativeTokenConfig.IsMintable {
```

Comment:
Stake info should be populated in case non-mintable native token is used, but stake token is different than native token.

We have opened a Pandora box regarding so many use cases with L1 vs L2 native tokens and stake token not being necesarily native one :smile:

Reply:
Let's address this through a separate PR.